src/OmniSharp.DotNetTest/OmniSharp.DotNetTest.csproj

```diff
@@ -16,7 +16,16 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.0.0" />
+  </ItemGroup>
+
+  <!-- Legacy 'dotnet test' support -->
+  <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Testing.Abstractions" Version="1.0.0-preview2-1-003177" />
```

Comment:
Yup. This is totally just used for a handful of exchange types with the old protocol. I'm considering just copying them in to remove this reference.